client/coral-admin/src/routes/Moderation/containers/Moderation.js

```diff
@@ -190,7 +190,7 @@ class ModerationContainer extends Component {
   };
 
   render () {
-    const {root, root: {asset, settings}, data} = this.props;
+    const {root, root: {asset}, data, settings} = this.props;
```

Comment:
^ Let's change it back an use `network-only` fetchPolicy as discussed in the Arch meeting